pkg/sarif_test.go

```diff
@@ -48,13 +82,14 @@ func TestSARIFOutput(t *testing.T) {
 		panic(fmt.Errorf("time.Parse: %w", e))
 	}
 
+	checkDocs, _ := sarifMockDocRead()
```

Comment:
Remove the `error` in func return if we don't need it?

Reply:
we need it to respect the interface. For the unit tests, I'm not using it because it never fails. I'll add a comment or check for the error. You're right it's confusing.

Reply:
added a comment to explain why I'm not catching the error.